mutt/signal.c

```diff
@@ -63,6 +63,9 @@ static sig_handler_t ExitHandler = mutt_sig_exit_handler;
 /// Function to handle SIGSEGV (11) signals
 static sig_handler_t SegvHandler = mutt_sig_exit_handler;
 
+/// Keep the old SEGV Handler, it might be have been set by ASAN
```

Comment:
Lowercase "handler"?

And I think gahr pointed out "it might **be** have been" on IRC.

Reply:
done